### DISCUSSION TITLE
Anything that can fix this connection issue?

### DISCUSSION THREAD

**Discussion by hockinsk** [jackaudio/jack2#925] [Q&A]
About to admit defeat on this. Trying to get a stable connection but failing at the following point. I do get connections, then they disappear, then return, then the application crashes. This is Dolby Atmos Renderer free trial available on Dolby's website.

> JackWinSemaphore::TimedWait name = jack_pipe.default_freewheel time_out
> JackFreewheelDriver::ProcessSync: SuspendRefNum error
> JackAudioDriver::ProcessGraphSync: ProcessWriteSlaves error, engine may now behave abnormally!!
> JackWinSemaphore::TimedWait name = jack_pipe.default_system time_out
> JackAudioDriver::ProcessGraphSync: SuspendRefNum error, engine may now behave abnormally!!
> 17:43:15.440 JACK connection graph change.
> CheckRes error
> Could not write notification
> ClientNotify fails name = Dolby Atmos Renderer notification = 12 val1 = 5 val2 = 1
> CheckRes error
> Could not write notification
> ClientNotify fails name = Dolby Atmos Renderer notification = 12 val1 = 6 val2 = 2

**Comment by SynthiaPayne** [comment]
One thought is that On Windows I have found that disabling ‘Exclusive Mode’ in the Preferences-Advanced Sound Control Panel settings fixed many connection and functionality issues. On Mar 24, 2023, at 10:49 AM, hockinsk ***@***.***> wrote:﻿
About to admit defeat on this. Trying to get a stable connection but failing at the following point. I do get connections, then they dissapear, then return, then the application crashes. This is Dolby Atmos Renderer free trial available on Dolby's website.
JackWinSemaphore::TimedWait name = jack_pipe.default_freewheel time_out JackFreewheelDriver::ProcessSync: SuspendRefNum error JackAudioDriver::ProcessGraphSync: ProcessWriteSlaves error, engine may now behave abnormally!! JackWinSemaphore::TimedWait name = jack_pipe.default_system time_out JackAudioDriver::ProcessGraphSync: SuspendRefNum error, engine may now behave abnormally!! 17:43:15.440 JACK connection graph change. CheckRes error Could not write notification ClientNotify fails name = Dolby Atmos Renderer notification = 12 val1 = 5 val2 = 1 CheckRes error Could not write notification ClientNotify fails name = Dolby Atmos Renderer notification = 12 val1 = 6 val2 = 2
